tests/rust/constant.rs

```diff
@@ -1,6 +1,9 @@
 const FOO: i32 = 10;
 const BAR: &'static str = "hello world";
 pub const DELIMITER: char = ':';
+pub const QUOTE: char = '\'';
+pub const TAB: char = '\t';
+pub const NEWLINE: char = '\n';
```

Comment:
Can you add some Unicode char examples too?

Reply:
I could but thought to do this step by step. The docu has this nice '❤' character.
https://doc.rust-lang.org/std/primitive.char.html#method.escape_unicode
Maybe escape_default does all this for free.

Reply:
To clarify - we probably need at least two examples here, with different handling:
 - 1) Any character with code point <= 0xFFFF - fits into `wchar_t` and should be written out as such.
 - 2) Any character with large code point - either produces a multichar or at least errors out.

Reply:
It seems better to fix the issue at once (especially since this PR will automatically close it when merged).

Unfortunately, `escape_default` uses a different escaping than C/C++ so you might still need to do something manually here.